tsconfig.json

```diff
@@ -15,5 +15,5 @@
     "types": ["node", "jest"]
   },
   "include": ["./src"],
-  "exclude": ["./boilerplate", "./templates"]
+  "exclude": ["./boilerplate", "./templates", "**/**.test.ts"]
```

Comment:
Test files in `src` were getting included in the build folder, and then run as duplicates during `yarn test`